executor/join.go

```diff
@@ -92,9 +93,7 @@ type HashJoinExec struct {
 	prepared    bool
 	isOuterJoin bool
 
-	// joinWorkerWaitGroup is for sync multiple join workers.
-	joinWorkerWaitGroup sync.WaitGroup
-	finished            atomic.Value
+	finished atomic.Value
```

Comment:
In go1.19, We have [atomic.Bool](https://pkg.go.dev/sync/atomic#Bool)

Reply:
Done.